SmartDeviceLinkTests/DevAPISpecs/SDLLockScreenManagerSpec.m

```diff
@@ -48,7 +48,7 @@ @interface SDLLockScreenManager ()
         
         it(@"should set properties correctly", ^{
             // Note: We can't check the "lockScreenPresented" flag on the Lock Screen Manager because it's a computer property checking the window
-            expect(fakePresenter.presented).to(beFalse());
+            expect(fakePresenter.presented).toEventually(beFalse());
```

Comment:
I tested keeping these as `to` and they still work. Why were they moved to `toEventually`?

Reply:
They are failing for me if I keep them as `to`s. The old code did not wait for the main thread, but the new code does. Maybe it depends on how fast your computer is?

Reply:
Odd. It shouldn't. `to`s are synchronous so it shouldn't be dependent on the speed of the computer.

Reply:
They fail on Travis too if I use `to`

Reply:
Weird. I don't know why it worked for me then.